src/qml/filters/audio_lowpass/ui.qml

```diff
@@ -54,9 +69,17 @@ Item {
         }
         Shotcut.Preset {
             id: preset
-            parameters: ['0', '1', '2', 'wetness']
-            Layout.columnSpan: 2
-            onPresetSelected: setControls()
+            parameters: ['0', '1', 'wetness']
+            Layout.columnSpan: parent.columns - 1
+            onBeforePresetLoaded: {
+                filter.resetProperty('0')
+                filter.resetProperty('1')
+                filter.resetProperty('wetness')
```

Comment:
You should replace these lines with `resetSimpleKeyframes()` from `KeyframableFilters`. Semantically they are not all simple keyframes in this filter, but it does the same thing by iterating over `keyframableParameters`.